src/main/java/org/broadleafcommerce/menu/processor/MenuProcessor.java

```diff
@@ -63,10 +61,7 @@ public int getPrecedence() {
     }
 
     @Override
-    public Map<String, Object> populateModelVariables(String tagName, Map<String, String> tagAttributes, BroadleafTemplateContext context) {
-        String resultVar = tagAttributes.get("resultVar");
-        String menuName = tagAttributes.get("menuName");
-        String menuId = tagAttributes.get("menuId");
+    public Map<String, Object> getMenu(String menuId, String menuName) {
```

Comment:
why do we need map as return type ? The idea of that method to get list of menu item dtos and put it under expected name in the map. In the new approach we don't use map we simply return what we ask for, in this case List<MenuItemDTO>